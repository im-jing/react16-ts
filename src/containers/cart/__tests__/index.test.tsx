import React from "react";
import { render, fireEvent, getByRole } from "@testing-library/react";
import { renderHook } from "@testing-library/react-hooks";

import "./matchMedia.mock";
import Cart from "../index";

describe("<Cart />", () => {
  it("render should match snapshot", () => {
    const { container, debug } = render(<Cart />);

    expect(container).toMatchSnapshot();
  });
});
