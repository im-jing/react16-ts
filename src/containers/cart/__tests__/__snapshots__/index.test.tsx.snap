// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Cart /> render should match snapshot 1`] = `
<div>
  <div
    class="content"
  >
    <div
      class="ant-table-wrapper"
    >
      <div
        class="ant-spin-nested-loading"
      >
        <div
          class="ant-spin-container"
        >
          <div
            class="ant-table"
          >
            <div
              class="ant-table-container"
            >
              <div
                class="ant-table-content"
              >
                <table
                  style="table-layout: auto;"
                >
                  <colgroup>
                    <col
                      class="ant-table-selection-col"
                    />
                    <col
                      style="width: 60%; min-width: 60%;"
                    />
                    <col
                      style="width: 40%; min-width: 40%;"
                    />
                  </colgroup>
                  <thead
                    class="ant-table-thead"
                  >
                    <tr>
                      <th
                        class="ant-table-cell ant-table-selection-column"
                      >
                        <div
                          class="ant-table-selection"
                        >
                          <label
                            class="ant-checkbox-wrapper"
                          >
                            <span
                              class="ant-checkbox"
                            >
                              <input
                                class="ant-checkbox-input"
                                type="checkbox"
                                value=""
                              />
                              <span
                                class="ant-checkbox-inner"
                              />
                            </span>
                          </label>
                        </div>
                      </th>
                      <th
                        class="ant-table-cell"
                        style="text-align: left;"
                      >
                        Title
                      </th>
                      <th
                        class="ant-table-cell"
                        style="text-align: right;"
                      >
                        Price
                      </th>
                    </tr>
                  </thead>
                  <tbody
                    class="ant-table-tbody"
                  >
                    <tr
                      class="ant-table-row ant-table-row-level-0"
                      data-row-key="1"
                    >
                      <td
                        class="ant-table-cell ant-table-selection-column"
                      >
                        <label
                          class="ant-checkbox-wrapper"
                        >
                          <span
                            class="ant-checkbox"
                          >
                            <input
                              class="ant-checkbox-input"
                              type="checkbox"
                              value=""
                            />
                            <span
                              class="ant-checkbox-inner"
                            />
                          </span>
                        </label>
                      </td>
                      <td
                        class="ant-table-cell"
                        style="text-align: left;"
                      >
                        car
                      </td>
                      <td
                        class="ant-table-cell"
                        style="text-align: right;"
                      >
                        100000
                      </td>
                    </tr>
                    <tr
                      class="ant-table-row ant-table-row-level-0"
                      data-row-key="2"
                    >
                      <td
                        class="ant-table-cell ant-table-selection-column"
                      >
                        <label
                          class="ant-checkbox-wrapper"
                        >
                          <span
                            class="ant-checkbox"
                          >
                            <input
                              class="ant-checkbox-input"
                              type="checkbox"
                              value=""
                            />
                            <span
                              class="ant-checkbox-inner"
                            />
                          </span>
                        </label>
                      </td>
                      <td
                        class="ant-table-cell"
                        style="text-align: left;"
                      >
                        shoes
                      </td>
                      <td
                        class="ant-table-cell"
                        style="text-align: right;"
                      >
                        500
                      </td>
                    </tr>
                    <tr
                      class="ant-table-row ant-table-row-level-0"
                      data-row-key="3"
                    >
                      <td
                        class="ant-table-cell ant-table-selection-column"
                      >
                        <label
                          class="ant-checkbox-wrapper"
                        >
                          <span
                            class="ant-checkbox"
                          >
                            <input
                              class="ant-checkbox-input"
                              type="checkbox"
                              value=""
                            />
                            <span
                              class="ant-checkbox-inner"
                            />
                          </span>
                        </label>
                      </td>
                      <td
                        class="ant-table-cell"
                        style="text-align: left;"
                      >
                        desk
                      </td>
                      <td
                        class="ant-table-cell"
                        style="text-align: right;"
                      >
                        2500
                      </td>
                    </tr>
                    <tr
                      class="ant-table-row ant-table-row-level-0"
                      data-row-key="4"
                    >
                      <td
                        class="ant-table-cell ant-table-selection-column"
                      >
                        <label
                          class="ant-checkbox-wrapper"
                        >
                          <span
                            class="ant-checkbox"
                          >
                            <input
                              class="ant-checkbox-input"
                              type="checkbox"
                              value=""
                            />
                            <span
                              class="ant-checkbox-inner"
                            />
                          </span>
                        </label>
                      </td>
                      <td
                        class="ant-table-cell"
                        style="text-align: left;"
                      >
                        book
                      </td>
                      <td
                        class="ant-table-cell"
                        style="text-align: right;"
                      >
                        50
                      </td>
                    </tr>
                  </tbody>
                </table>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      aria-live="polite"
      class="total-container"
    >
      Total: ï¿¥
      <span>
        0
      </span>
    </div>
  </div>
</div>
`;
